serverside/open-database-connection.php

```diff
@@ -1,9 +1,9 @@
 <?php
 
 $servername = "localhost";
-$username = "dphlqsmy_byfo-user0";
-$password = "06d%NA-xd3m%";
-$database = "dphlqsmy_byfo2020";
+$username = "mamp";
+$password = "YtC7W0MK52rZiRKT";
+$database = "byfo2020";
```

Comment:
Oops, that one shouldn't be checked in like this either because then it won't work on the server.  Maybe there's a way we can have an ignored config file that overrides the defaults...?